helm-charts/signals-gisib-ams/values.yaml

```diff
@@ -0,0 +1,102 @@
+tags:
+  postgresql: true
+  rabbitmq: true
+
+replicaCount: 1
+
+image:
+  pullPolicy: IfNotPresent
+  repository:
+  tag:
+
+podAnnotations: {}
+
+podSecurityContext:
+   fsGroup: 999
+
+securityContext: {}
+
+service:
+  type: ClusterIP
+  port: 80
+
+ingress:
+  enabled: true
+  className: ""
+  annotations: {}
+  hosts:
+    - host: gisib.signals.local
+      paths:
+        - path: /
+          pathType: ImplementationSpecific
+  tls: []
+
+resources: {}
+
+nodeSelector: {}
+
+tolerations: []
+
+affinity: {}
+
+existingSecret: my-custom-signals-gisib-ams-secrets  # If set to null the secrets are managed by Helm
```

Comment:
I would set this to null by default, so a Helm install command without any values works by default.